notificationworkerlambda/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="net.logstash.logback.encoder.LogstashEncoder">
+            <pattern>
+                %d{yyyy-MM-dd HH:mm:ss} [%t] %X{AWSRequestId} %-5p %c{1}:%L - %m%n
+            </pattern>
+        </encoder>
+    </appender>
+    <root level="info">
```

Comment:
When the level was "all" we got a lot of trace log data, which I believe cluttered up our view of what was happening in our lambdas. For now, I believe this level logs logger.info and logger.error statements, so should be equivalent to our previous log configuration.